py/surveyqa/summary.py

```diff
@@ -482,38 +501,39 @@ def makeplots(exposures, tiles, outdir):
 
     template += """
         <div class="flex-container">
-            <div class="column side"></div>          
+            <div class="column side"></div>
             <div class="column middle">
                 <div class="header">
                     <p class="sansserif">Progress Graphs</p>
-                </div> 
-                
+                </div>
+
                 <div class="flex-container">
                     <div>{{ skyplot_script }} {{ skyplot_div }}</div>
                     <div>{{ progress_script }} {{ progress_div }}</div>
                     <div>{{ progress_script_1 }} {{ progress_div_1 }}</div>
-                </div>    
-                
+                    <div>{{ moonplot_script }} {{ moonplot_div }}</div>
+                </div>
+
```

Comment:
This moonplot got added as part of the "progress graphs" section, which seems misplaced.  It fits more with the histograms, while renaming that section to something like "Observing Conditions".

@ana-lyons please confirm that this would be consistent with any updates that you are working on to avoid merge conflicts.

Reply:
as it is right now, it looks fine with what I have. should I move the moonplot graph to the histograms section and rename it? or should William do that.

Reply:
Thanks for the confirmation @ana-lyons.  Let's move the div and rename the section within this branch before merging.  @williyamshoe can do that as part of the other requested updates in this PR.

Reply:
Alright, will do. I wasn't sure where to put the plot since I felt it didn't fit in the progress nor the histogram section. I'll move it to the histogram section and rename the section to "Observing Conditions", along with committing the other changes listed above.